core-it-suite/src/test/java/org/apache/maven/it/MavenITmng7470ResolverTransportTest.java

```diff
@@ -105,14 +105,7 @@ private void performTest( /* nullable */ final String transport, final String lo
     public void testResolverTransportDefault()
             throws Exception
     {
-        if ( matchesVersionRange( "[4.0.0-alpha-3,)" ) )
-        {
-            performTest( null, NATIVE_LOG_SNIPPET );
-        }
-        else // [3.9.0,) but not 4, see above and ctor
-        {
-            performTest( null, WAGON_LOG_SNIPPET );
```

Comment:
Can this snippet be deleted then?

Reply:
No, as this class has 3 tests: default, forced-wagon, forced-native and is used in forced-wagon test. This change is only about default one (nothing forced).